components/sankey/events.md

```diff
@@ -0,0 +1,517 @@
+---
+title: Events
+page_title: Sankey Diagram Events
+description: Events of the Sankey Diagram for Blazor.
+slug: sankey-events
+tags: telerik,blazor,sankey,diagram,chart,events
+published: True
+position: 9
+---
+
+# Sankey Diagram Events
```

Comment:
Did we abandon the idea for a single example in this article?

Reply:
The idea still stands but I can do it as part of the data refactoring after the release.